app/pages/dashboard/contextCreator.js

```diff
@@ -9,4 +10,5 @@ export const getContext = ({ organisations }) => ({
       { data: row.odsCode || '' },
     ])
     : [],
+  backLinkHref: `${baseUrl}/`,
```

Comment:
😢 I don't think this is right here. From the dashboard we click back we want to go back to public-browse.

Reply:
OH YEH GOOD SPOT 👀

Reply:
Leave this as is. We'll come back to this in a separate ticket.

Reply:
I have changed it to '/' as `/admin/` doesn't go anywhere

Reply:
I have changed it to '/' as '/admin/' doesn't go anywhere